bin/compile-requirements.sh

```diff
@@ -6,8 +6,8 @@ export CUSTOM_COMPILE_COMMAND="make compile-requirements"
 
 # We need this installed, but we don't want it to live in the main requirements
 # We will need to periodically review this pinning
-pip install --upgrade "pip<22"  # Temporary until pip-tools is fixed.
-pip install --upgrade pip-tools==6.4.0  # Needs at least this version to build.
+pip install --upgrade pip
```

Comment:
We should prob keep it pinned to a version that works, in case future pip breaks it

```suggestion
pip install --upgrade pip==22.0
```

Reply:
Since this command is a manual thing I think this is okay. We will notice and investigate if there's breakage.